cel/env.go

```diff
@@ -204,12 +204,8 @@ type env struct {
 
 // Check implements the Env interface method.
 func (e *env) Check(ast Ast) (Ast, Issues) {
-	pe, err := AstToParsedExpr(ast)
-	if err != nil {
-		errs := common.NewErrors(ast.Source())
-		errs.ReportError(common.NoLocation, err.Error())
-		return nil, &issues{errs: errs}
-	}
+	// Note, errors aren't currently possible on the Ast to ParsedExpr conversion.
```

Comment:
If this is expected to change ~soon, let's link the github issue for it. If it's a note about a design limitation, it should be OK to leave as-is.

Reply:
Yep, this is just a design note. For the foreseeable future, it's not possible to produce an error when converting between an `Ast` and a `ParsedExpr`, but it is possible with a `CheckedExpr`. In order to keep the interface symmetric there's an `error` on the return.